### ISSUE TITLE
Improve Contributing.md

### ISSUE THREAD

**Issue by caseyprovost** [maxfierke/resumis#38]
Currently the app is multi-tenant but there are no instructions in the readme about how to support custom domains locally while contributing. There should also be an outline of how to setup the app with explanations, standards on how to contribute, and even perhaps a code of conduct.

**Comment by maxfierke** [comment]
Added a code of conduct and file on contributing. I'll address some of the other documentation related to multi-tenancy and deployment in some wiki pages this week, hopefully. Currently finishing up production-ization of some Docker stuff and will write up a guide on that.

I'm currently debating whether to keep multi-tenancy around (leaning more towards keeping it), so I removed some stuff about it in the README until I decide, but here are some pointers for posterity:

* In dev, you'll want to use `lvh.me` or another locally resolving domain name with support for wildcard subdomains. I've used [Anvil](http://anvilformac.com/) and [Pow](http://pow.cx/) in the past, which handle setting applications up under local *.dev domains (or other TLDs). Can also roll one's own with `dnsmasq` or `launchdns` (on macOS) and have a locally resolving TLD that way.

Set these two ENV vars (I've used .rbenv-vars in the past for this, but should probably just use `dotenv-rails`):

* `RESUMIS_CANONICAL_HOST` - Bare domain of the resumis installation for multi-tenancy mode. Canonical hostname for single-tenancy mode.
  * e.g. a multi-tenant resumis host would use `resumis.com`
  * a single-tenant host might use `www.maxfierke.com`
* `RESUMIS_TENANCY_MODE` - Can be `single` or `multi`. **This is not easily changed later**
